modal_proto/api.proto

```diff
@@ -2368,6 +2368,17 @@ message SystemErrorMessage {
   string error_message = 2;
 }
 
+message TaskClusterHelloRequest {
+  string task_id = 1;
+  string container_ip = 2;
+}
+
+message TaskClusterHelloResponse {
```

Comment:
cluster members are started at the same time but they won't execute this RPC at the same time, and it seems plausible that the last hello could arrive >60s after the first hello, causing an RPC timeout because not all `container_ips` are available.

If this is a problem in practice, the server could stop polling redis after `timeout - 5s` and return `FAILED_PRECONDITION` or some other retry-able status when not all members say hello in time, or we could split out "hello" from a new "get cluster IPs" RPC.

Reply:
> If this is a problem in practice

I don't think it should be -- 60 seconds is more than enough.

Currently, the server implementation stops polling Redis after ~55 seconds, then returns `DEADLINE_EXCEEDED` and the client will retry.